editor.planx.uk/src/pages/FlowEditor/components/Settings/Submissions/EventsLog.tsx

```diff
@@ -121,6 +139,24 @@ const CollapsibleRow: React.FC<Submission> = (submission) => {
           {format(new Date(submission.createdAt), "dd/MM/yy hh:mm:ss")}
         </TableCell>
         <TableCell>{submission.sessionId}</TableCell>
+        <TableCell>
+          {showDownloadButton && (
+            <IconButton
+              aria-label="download application"
+              size="small"
+              onClick={() => {
+                const zipUrl = `${
+                  import.meta.env.VITE_APP_API_URL
+                }/download-application-files/${
+                  submission.sessionId
+                }?localAuthority=${teamSlug}&email=${submissionEmail}`;
+                window.open(zipUrl, "_blank");
```

Comment:
This is really simple! Is `window.open()` actually _too_ simple / let's get council folks to test on staging to confirm that their council browser settings won't block this? 

We also know that we want to "harden" `/download-application-files/:sessionId` endpoint in the near future ([this Trello ticket](https://trello.com/c/EeJb0QVC/2614-harden-download-application-files-endpoint)) - so we'll ideally want a solution that works for both places here so that we can ensure send to email & submission log zips are totally consistent for end uers ! I've attached this PR to the Trello ticket for future reference.

Reply:
Only one way to find out! This is the approach I would have gone with initially also 👍